scripts/babbage/alonzo-babbage-test-genesis.json

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
Why do you need this file?

Reply:
because the current `cardano-testnet` for Babbage looks for it locally, see https://github.com/input-output-hk/cardano-node/blob/master/cardano-testnet/src/Testnet/Babbage.hs#L97

Reply:
:+1: